src/pages/LoginProcess.tsx

```diff
@@ -1,62 +1,47 @@
+/* eslint-disable max-len */
 import { useEffect } from 'react';
-
 import { useLocation, useNavigate } from 'react-router-dom';
-
 import useLoginUser from '../hooks/useFetchUserData';
 import { useAuth } from '../context/AuthContext';
 import { Endpoints } from '../constants/Endpoints';
+import Spinner from '../components/Spinner/Spinner';
 
-/**
- * Temporary login process page shown as user is logging in.
- */
 const LoginProcessPage = () => {
-	// context for handling user data
 	const { setUserData, setIsLoggedIn } = useAuth();
-
-	// handles navigation
 	const navigate = useNavigate();
-
-	// used in retrieving query params
 	const location = useLocation();
 
-	// retrieve provider and key from query params to be used for getting user data
+	// Retrieve provider and key from query params to be used for getting user data
```

Comment:
Unless comments are being written for a function, we don't begin with a capital letter across the codebase